app/parse.py

```diff
@@ -1,4 +1,11 @@
+import requests
 from dataclasses import dataclass
+from bs4 import BeautifulSoup
+
+BASE_URL = "https://mate.academy/"
+SELECTOR_NAME = ".ProfessionCard_title__Zq5ZY"
+SELECTOR_DESCRIPTION = ".typography_landingTextMain__Rc8BD.mb-32"
+SELECTOR_DURATION = ".ProfessionCardTags_regularTag__yTc6K"
```

Comment:
Verify that the CSS selectors (`SELECTOR_NAME`, `SELECTOR_DESCRIPTION`, `SELECTOR_DURATION`) match the actual HTML structure of the target page. If the structure changes, these selectors may need to be updated.